### PR TITLE
chore: update to Rust 1.75

### PR DESCRIPTION
**Pull request by linbingquan**
chore: update to Rust 1.75

I am not sure `cacheVersion` should be updated.

If this PR has any problem, please tell me.

### CODE REVIEW

**Comment by bartlomieju** [comment]
> I am not sure cacheVersion should be updated.

Let's do it for good measure 👍

File: cli/args/mod.rs [hunk 1]
```diff
@@ -17,9 +17,9 @@ use deno_semver::npm::NpmPackageReqReference;
 use indexmap::IndexMap;
 
 pub use deno_config::BenchConfig;
-pub use deno_config::CompilerOptions;
+
 pub use deno_config::ConfigFile;
-pub use deno_config::EmitConfigOptions;
+
```
Comment (bartlomieju) [inline cli/args/mod.rs:22]: Could you cleanup these leftover empty lines?

File: cli/cache/mod.rs [hunk 2]
```diff
@@ -45,7 +45,7 @@ pub use disk_cache::DiskCache;
 pub use emit::EmitCache;
 pub use incremental::IncrementalCache;
 pub use module_info::ModuleInfoCache;
-pub use module_info::ModuleInfoCacheModuleAnalyzer;
+
```
Comment (bartlomieju) [inline cli/cache/mod.rs:48]: Ditto

File: cli/tsc/mod.rs [hunk 3]
```diff
@@ -50,7 +50,7 @@ mod diagnostics;
 
 pub use self::diagnostics::Diagnostic;
 pub use self::diagnostics::DiagnosticCategory;
-pub use self::diagnostics::DiagnosticMessageChain;
+
```
Comment (bartlomieju) [inline cli/tsc/mod.rs:53]: Ditto

File: cli/args/mod.rs (hunk 1 above)
Comment (linbingquan) [reply cli/args/mod.rs]: Sorry, my mistake.

**Review by bartlomieju — APPROVED** [review]
Sweet, thank you @linbingquan!

**Pull request merged**